src/test/controller/UserController.test.ts

```diff
@@ -222,9 +222,57 @@ describe('Unit tests for UserController', function () {
       });
   });
 
-  test('DELETE /users/:id', async (done) => {
+  test('DELETE /users/:id - ForbiddenError should be caught', async (done) => {
     const expectedUser: User = new UserFactory().generate();
+    expectedUser.firebaseId = 'randomfirebaseId';
+
+    const authenticatedUser: User = new UserFactory().generate();
+    authenticatedUser.firebaseId = firebaseId;
+
+    await userRepository.save(expectedUser);
+    await userRepository.save(authenticatedUser);
+
+    request(app)
+      .delete(`/users/${expectedUser.id}`)
+      .set('Authorization', idToken)
+      .expect(403)
+      .end((err, res) => {
+        const frontEndErrorObject = res.body;
+        expect(frontEndErrorObject?.errorCode).toEqual('ForbiddenError');
+        expect(frontEndErrorObject.message).toHaveLength(1);
+        expect(frontEndErrorObject.message[0]).toEqual(
+          'Your account does not have sufficient privileges to perform this action.'
+        );
+        done();
+      });
+  });
```

Comment:
I was thinking this might make it easier for the developer to understand which user is which. What do you think?

```suggestion
    const otherUser: User = new UserFactory().generate();
    otherUser.firebaseId = 'randomfirebaseId';

    const userWhoWantsToDeleteOtherUser: User = new UserFactory().generate();
    userWhoWantsToDeleteOtherUser.firebaseId = firebaseId;

    await userRepository.save(otherUser);
    await userRepository.save(userWhoWantsToDeleteOtherUser);

   // simulating user who is making request to delete another use
    request(app)
      .delete(`/users/${otherUser.id}`)
      .set('Authorization', idToken)
      .expect(403)
      .end((err, res) => {
        const frontEndErrorObject = res.body;
        expect(frontEndErrorObject?.errorCode).toEqual('ForbiddenError');
        expect(frontEndErrorObject.message).toHaveLength(1);
        expect(frontEndErrorObject.message[0]).toEqual(
          'Your account does not have sufficient privileges to perform this action.'
        );
        done();
      });
  });
```

Reply:
Hmmm, how about `deletingUser` and `authenticatingUser`?

Reply:
Whichever you think makes it more clear!